apps/frontend/src/network/profile/__tests__/Routes.test.tsx

```diff
@@ -144,4 +145,40 @@ describe('with team', () => {
       ]
     `);
   });
+
+  describe('staff', () => {
+    it('calculates links', async () => {
+      nock.cleanAll();
+      nock(API_BASE_URL, {
+        reqheaders: { authorization: 'Bearer token' },
+      })
+        .get('/users/43')
+        .reply(200, {
+          ...user,
+          id: '43',
+          teams: [
+            {
+              id: '0',
+              displayName: 'ASAP',
+              role: 'Staff',
+              approach: 'approach',
+              responsibilities: 'responsible',
+            },
+          ],
+          role: 'Staff',
+        } as UserResponse);
+
+      const { queryAllByRole, getByText } = await renderProfile('43');
+      const loadingIndicator = getByText(/loading/i);
+      await waitForElementToBeRemoved(loadingIndicator);
+      const links = (await queryAllByRole('link')) as HTMLAnchorElement[];
+      expect(links.map(({ href }) => href)).toMatchInlineSnapshot(`
+        Array [
```

Comment:
These kinds of tests are getting a bit out of hand in the FE, the test above you copied this from is quite possibly the worst test in the frontend.
I would suggest having a case for the render condition (that looks for some `'from ASAP'` rendered by `Staff`) and then a test that gets one of the links and checks its href